### PR TITLE
Xml cleanup, part 1

### PR DESCRIPTION
**Pull request by andrewdotn**
Remove some code that’s now unused.

### CODE REVIEW

File: src/CreeDictionary/CreeDictionary/paradigm/manager.py [hunk 1]
```diff
@@ -52,6 +52,9 @@ def paradigm_for(
         layout_sizes = self._layout_sizes_or_raise(paradigm_name)
         if size is None:
             size = self.default_size(paradigm_name)
+
+        if size not in layout_sizes:
+            raise ParadigmDoesNotExistError()
```
Comment (eddieantonio) [inline src/CreeDictionary/CreeDictionary/paradigm/manager.py:57]:
```suggestion
            raise ParadigmDoesNotExistError(f"size {size!r} for {paradigm_name}")
```

File: src/CreeDictionary/CreeDictionary/views.py [hunk 2]
```diff
@@ -346,46 +345,24 @@ def should_include_auto_definitions(request):
...
     """
-    # TODO: make this use the new-style ParadigmManager exclusively
 
     manager = default_paradigm_manager()
 
-    if isinstance(paradigm_size, ParadigmSize):
-        size = convert_crkeng_paradigm_size_to_size(paradigm_size)
-    else:
-        size = paradigm_size
+    size = paradigm_size
```
Comment (eddieantonio) [inline src/CreeDictionary/CreeDictionary/views.py:357]: Is there an issue with changing the argument name of this function to get rid of this alias?

File: src/CreeDictionary/tests/test_fst_regressions.py [hunk 3]
```diff
@@ -1,22 +1,18 @@
...
 
 
 def test_fst_analysis():
-    analyses = set(temp_hfstol.analyze("ta-pe-kiwemakaniyiw"))
+    analyses = rich_analyze_relaxed("ta-pe-kiwemakaniyiw")
     assert (
-        Analysis(
-            raw_prefixes="PV/ta+PV/pe", lemma="kîwêmakan", raw_suffixes="V+II+Ind+4Sg"
-        )
+        RichAnalysis((("PV/ta+", "PV/pe+"), "kîwêmakan", ("+V", "+II", "+Ind", "+4Sg")))
```
Comment (eddieantonio) [inline src/CreeDictionary/tests/test_fst_regressions.py:11]: Ahhhhh, this is so much cleaner!

**Review by eddieantonio — APPROVED** [review]
🤩 🤩 🤩  BEAUTIFUL!

File: src/CreeDictionary/CreeDictionary/views.py (hunk 2 above)
Comment (andrewdotn) [reply src/CreeDictionary/CreeDictionary/views.py:357]: No problem, just didn’t notice that.

**Pull request merged**